include/cross.h

```diff
@@ -161,4 +161,13 @@ bool get_expanded_files(const std::string &path,
 
 std::string get_language_from_os();
 
+#ifndef WIN32
+#define XATTR_FATTR_NAME "user.dosbox_fattr"
```

Comment:
- Please only use `const` or `constexpr`.
- `const char*` is fine for string constants, otherwise default to `std::string`.
- Please use UpperCamelCase for new constants and convert old constants if you touch related code.
- This name is too cryptic, this would be better:

```cpp
constexrp char DosAttrExtendedAttributeName[] = "user.dosbox_fattr"
```

Reply:
This is 31 flipping characters, it doesn't need to be that long. FatAttribsXAttrName is fine, xattr is an official name for that feature used in a bunch of places.